projects/js-packages/connection/changelog/update-rna-connect-button

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Should this be a major change, since we're changing the name of components and that could be a breaking change for some folks?

Reply:
Good catch, fixed!